Client/Frontend/Browser/DomainUserScript.swift

```diff
@@ -58,6 +59,8 @@ enum DomainUserScript: CaseIterable {
                  "talk.bravesoftware.com", "beta.talk.bravesoftware.com",
                  "dev.talk.brave.software", "beta.talk.brave.software",
                  "talk.brave.software"])
+    case .bravePlaylistFolderSharingHelper:
+      return Set(["playlist.bravesoftware.com", "playlist.brave.com", "127.0.0.1"])
```

Comment:
Could be nice to add a comment as a reminder to remove when server is ready

Reply:
I will have to change all of this anyway once I get the real URL. So no worries here.